azure-toolkit-libs/azure-toolkit-appservice-lib/src/main/java/com/microsoft/azure/toolkits/appservice/entity/WebAppDeploymentSlotEntity.java

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+
+package com.microsoft.azure.toolkits.appservice.entity;
+
+import com.azure.resourcemanager.appservice.models.DeploymentSlot;
+import com.microsoft.azure.toolkits.appservice.model.Runtime;
+import com.microsoft.azure.toolkits.appservice.utils.Utils;
+import lombok.Getter;
+import lombok.experimental.SuperBuilder;
+
+import java.util.Map;
+
+@Getter
+@SuperBuilder(toBuilder = true)
+public class WebAppDeploymentSlotEntity {
+    private String id;
+    private String name;
+    private String webappName;
+    private String resourceGroup;
+    private String subscriptionId;
+    private String appServicePlanId;
```

Comment:
extract common :
    private String id;
    private String name;
    private String webappName;
    private String resourceGroup;
    private String subscriptionId;

Reply:
May sync and do this later in common lib